### PR TITLE
Lab5

### PR DESCRIPTION
**Pull request by polinabogd**

### CODE REVIEW

**Review by yeldmitrenko — CHANGES_REQUESTED** [review]
Please fix my comments

File: lab5.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using namespace std;
+class Time{
+    public:
+    int hh, mm, ss;
+    void convert(int ss)
+    {
+        hh = trunc(ss/3600);
+        mm = trunc((ss%3600)/60);
+        ss = trunc(mm%60);
+        cout<<hh<<":"<<mm<<":"<<ss<<endl;
```
Comment (yeldmitrenko) [inline lab5.h:16]: You should rewrite it from "`cout<<hh<<":"<<mm<<":"<<ss<<endl;`" to "`cout << hh << ":" << mm << ":"<< ss << endl;`"

File: lab5.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+using namespace std;
+class Time{
+    public:
+    int hh, mm, ss;
+    void convert(int ss)
+    {
+        hh = trunc(ss/3600);
+        mm = trunc((ss%3600)/60);
+        ss = trunc(mm%60);
```
Comment (yeldmitrenko) [inline lab5.h:15]: Please make spaces on both sides of "%"

File: lab5.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        cout<<hh<<":"<<mm<<":"<<ss<<endl; 
+    }
+    void convert(int hh, int mm, int ss)
+    {
+        cout<<hh*3600 + mm*60 + ss<<endl;
```
Comment (yeldmitrenko) [inline lab5.h:20]: Please make it clear, what is it 3600 and what is it 60. It should be better to write constant at the beginning of the file. Moreover, please make spaces on both sides of "*" and add spaces between "cout", "<<", "hh" and "ss", "<<", "endl;"

**Review by yeldmitrenko — CHANGES_REQUESTED** [review]
Please fix my comments

File: lab5.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+#include <string>
+#include <cmath>
+#include <tuple>
+
+using namespace std;
+class Time{
+    public:
+    int hh, mm, ss;
+    void convert(int ss)
+    {
```
[the remark this answers is not in the document]
Comment (polinabogd) [reply lab5.h:12]: In this case i find it better to leave the implementation inside of the class.

File: lab5.h [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+#include <iostream>
+#include <math.h>
+#define findHours 3600
+#define findMinutes 60
+
+class Time{
+    public:
+        int hh, mm, ss;
+
+    void ConvertTime(int ss)
```
Comment (yeldmitrenko) [inline lab5.h]: Remove the class method to 1 tab to the right

File: lab5.h [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        hh = trunc(ss / findHours);
+        mm = trunc((ss % findHours) / findMinutes);
+        ss = trunc(mm % findMinutes);
+        cout << hh << ":" << mm << ":"<< ss << endl;
+    }
+
+    void ConvertTime(int hh, int mm, int ss)
```
Comment (yeldmitrenko) [inline lab5.h]: Remove the class method to 1 tab to the right

**Review by yeldmitrenko — APPROVED** [review]

**Pull request merged**